### PR TITLE
Clarified n-gon formulas.

### PR DESCRIPTION
**Pull request by bdice**
## Description
I clarified the formulas for n-gon generation. I found the previous code comments a little confusing, since they looked like dead code instead of a math expression.

## Types of Changes

- [x] Documentation update
- [x] Bug fix
- [ ] New feature
- [ ] Breaking change<sup>1</sup>

## Checklist:
- [x] I am familiar with the [**Contributing Guidelines**](https://github.com/glotzerlab/coxeter/blob/master/CONTRIBUTING.md).
- [x] I agree with the terms of the [**Contributor Agreement**](https://github.com/glotzerlab/coxeter/blob/master/ContributorAgreement.md).
- [x] The changes introduced by this pull request are covered by existing or newly introduced tests.
- [ ] I have updated the [changelog](https://github.com/glotzerlab/coxeter/blob/master/ChangeLog.txt) and the [credits](https://github.com/glotzerlab/coxeter/blob/master/doc/source/credits.rst).

### CODE REVIEW

File: coxeter/families/common.py [hunk 1]
```diff
@@ -52,20 +52,21 @@ def make_vertices(cls, n):
...
 
-        # Area of an n-gon inscribed in a circle
-        # A_poly = ((n*r**2)/2)*np.sin(2*np.pi/n)
-        # A_circ = np.pi*r**2
-        # pos *= np.sqrt(A_circ/A_poly)
-        a_circ_a_poly_sq = np.pi / ((n / 2) * np.sin(2 * np.pi / n))
-        pos *= np.sqrt(a_circ_a_poly_sq)
+        # The area of an n-gon inscribed in a circle is given by:
+        # \frac{n r^2}{2} \sin(2\pi / n)
+        # The ratio of that n-gon area to its inscribed circle area is:
```
Comment (vyasr) [inline coxeter/families/common.py:63]:
```suggestion
        # The ratio of that n-gon area to its circumscribed circle area is:
```

File: coxeter/families/common.py [hunk 2]
```diff
@@ -26,7 +26,7 @@ class RegularNGonFamily(ShapeFamily):
 
     @classmethod
     def get_shape(cls, n):
-        """Generate an n-gon.
+        """Generate an n-gon with area 1.
```
Comment (vyasr) [inline coxeter/families/common.py:29]:
```suggestion
        """Generate a unit area n-gon.
```
My personal preference, but also what's already used elsewhere in the docstrings.

File: coxeter/families/common.py [hunk 3]
```diff
@@ -39,7 +39,7 @@ def get_shape(cls, n):
 
     @classmethod
     def make_vertices(cls, n):
-        """Generate vertices of an n-gon.
+        """Generate vertices of an n-gon with area 1.
```
Comment (vyasr) [inline coxeter/families/common.py:42]:
```suggestion
        """Generate vertices of a unit area n-gon.
```

File: coxeter/families/common.py [hunk 4]
```diff
@@ -52,20 +52,21 @@ def make_vertices(cls, n):
...
-        # pos *= np.sqrt(A_circ/A_poly)
-        a_circ_a_poly_sq = np.pi / ((n / 2) * np.sin(2 * np.pi / n))
-        pos *= np.sqrt(a_circ_a_poly_sq)
+        # The area of an n-gon inscribed in a circle is given by:
+        # \frac{n r^2}{2} \sin(2\pi / n)
+        # The ratio of that n-gon area to its inscribed circle area is:
+        a_circ_a_poly = np.pi / ((n / 2) * np.sin(2 * np.pi / n))
 
-        return pos.T
+        # Rescale the positions so that the final shape has area 1
```
Comment (vyasr) [inline coxeter/families/common.py:66]:
```suggestion
        # Rescale the positions so that the final shape has area 1.
```

**Review by vyasr — APPROVED** [review]
Thanks! I'm going to apply my proposed changes then merge.

**Comment by bdice** [comment]
@vyasr I agree with all your suggested changes. Thanks!

**Pull request merged**